clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -738,22 +736,21 @@ the following style significantly:
   import M;
   ... // use declarations from module M.
 
-The key part of the tip is to reduce the duplications from the text includes.
+Reducing the duplication from textual includes is what improves compile-time
+performance.
 
-Ideas for converting to modules
--------------------------------
+Transitioning to modules
+------------------------
 
-For new libraries, we encourage them to use modules completely from day one if possible.
-This will be pretty helpful to make the whole ecosystems to get ready.
-
-For many existing libraries, it may be a breaking change to refactor themselves
-into modules completely. So that many existing libraries need to provide headers and module
+New code and libraries should use modules from day one if possible. However,
+it may be a breaking change for existing code or libraries to refactor to use
+modules. As a result, many existing libraries need to provide headers and module
 interfaces for a while to not break existing users.
-Here we provide some ideas to ease the transition process for existing libraries.
-**Note that the this section is only about helping ideas instead of requirement from clang**.
 
-Let's start with the case that there is no dependency or no dependent libraries providing
-modules for your library.
+This section provides some ideas on how to ease the transition process for
+existing libraries. **Note that this information is only intended as helpful
```

Comment:
```suggestion
existing libraries. **Note that this information is only intended as guidance, rather than as requirements to use modules in Clang.
```